cider-velox/src/CiderPlanNode.cpp

```diff
@@ -19,11 +19,11 @@
  * under the License.
  */
 
-#include "CiderPlanNode.h"
-
 #include <mutex>
 
+#include "CiderPlanNode.h"
 #include "CiderPlanNodeTranslator.h"
+#include "cider/CiderException.h"
```

Comment:
why need include this header here, seems not in use.

Reply:
revomed

Reply:
same as other files under `cider-velox` folder